internal/models/hype_train.go

```diff
@@ -0,0 +1,57 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package models
+
+type ContributionData struct{
+	TotalContribution   		 	 int64 	`json:"total"`
+	TypeOfContribution        		 string `json:"type"`
+	UserWhoMadeContribution	    	 string `json:"user_id"`
+	UserNameWhoMadeContribution	     string `json:"user_name"`
+	UserLoginWhoMadeContribution     string `json:"user_login"`
+}
```

Comment:
Since you use this for Websub, it isn't reflected properly since they use a singular `user` field for user_id. Could use the same strategy as below for `omitempty` or optionally just make another one just for WS.